server/server.go

```diff
@@ -577,6 +577,9 @@ func (s *Server) startReloadWatcher(ctx context.Context) {
 				logrus.Errorf("Unable to reload configuration: %v", err)
 				continue
 			}
+			// ImageServer compiles the list with regex for both
+			// pinned and sandbox/pause images, we need to update them
+			s.StorageImageServer().UpdatePinnedImagesList(append(s.config.PinnedImages, s.config.PauseImage))
```

Comment:
With this change, you can drop this code: https://github.com/cri-o/cri-o/blob/main/internal/storage/image.go#L857

Reply:
ah, right!